cmd/tailscale/cli/update.go

```diff
@@ -24,7 +37,7 @@ import (
 var updateCmd = &ffcli.Command{
 	Name:       "update",
 	ShortUsage: "update",
-	ShortHelp:  "Update Tailscale to the latest/different version",
+	ShortHelp:  "[ALPHA] Update Tailscale to the latest/different version",
```

Comment:
Have we used "Alpha" or other warnings in other parts of the CLI? Not afaik?
If not, then not sure we should include, unless we're going to add throughout the CLI.

Reply:
Yes, we have. I made it match the others:

```
bradfitz@tsdev:~/src/tailscale.com$ git grep ALPHA
cmd/tailscale/cli/login.go:     ShortUsage: "[ALPHA] login [flags]",
cmd/tailscale/cli/serve.go:             ShortHelp: "[ALPHA] Serve from your Tailscale node",
cmd/tailscale/cli/serve.go:*** ALPHA; all of this is subject to change ***
cmd/tailscale/cli/switch.go:  [ALPHA] switch <name>
cmd/tailscale/cli/switch.go:  [ALPHA] switch --list
cmd/tailscale/cli/update.go:    ShortHelp:  "[ALPHA] Update Tailscale to the latest/different version",
```